deploys/demo.syncfor.science/config/vendors/epic.yml

```diff
@@ -13,6 +13,7 @@ auth:
   sign_in_steps:
     - element: "#Login"
       send_keys: "fhirjason"
+      wait: 5
```

Comment:
Oh, thanks for clarifying that wait --> Python sleep. But still, implicitWait should be less fragile (and should save time when things are working well).